frontend/lib/src/mocks/arrow/types/timedelta.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Raw data (in Apache Arrow format) for a dataframe
```

Comment:
do we need this commented?

Reply:
Hmm, we have the same type of comment in all mock types... so for consistency,, it's probably good to keep it in. Especially, the specific python code that this is based on is important for reproducibility.